cmd/workflow-controller/main.go

```diff
@@ -96,6 +116,8 @@ func NewRootCommand() *cobra.Command {
 	command.Flags().IntVar(&glogLevel, "gloglevel", 0, "Set the glog logging level")
 	command.Flags().IntVar(&workflowWorkers, "workflow-workers", 8, "Number of workflow workers")
 	command.Flags().IntVar(&podWorkers, "pod-workers", 8, "Number of pod workers")
+	command.Flags().BoolVar(&namespaceMode, "namespace-mode", false, "run workflow-controller as namespace mode")
+	command.Flags().StringVar(&filteredNamespace, "namespace", "", "namespace that workflow-controller listens to, default to the installation namespace")
```

Comment:
Isn't --namespace already available because we added the kubectl flags to the command? I think this is redundant.

Reply:
This is for cmd `workflow-controller`, were kubectl flags added to it?

Reply:
`--watched-namespace`
`WATCHED_NAMESPACE`
```
command.Flags().StringVar(&watchedNamespace, "-watched-namespace", os.Getenv("WATCHED_NAMESPACE")
```